schema/project-level/project-schema.json

```diff
@@ -1010,7 +1010,7 @@
         },
         "roles": {
           "title": "Party roles",
-          "description": "The party's role(s) in the project. Role(s) should be taken from the [partyRole codelist](http://standard.open-contracting.org/1.1/en/schema/codelists/#party-role). Values from the provided codelist should be used wherever possible, though extended values can be provided if the codelist does not have a relevant code.",
+          "description": "The party's role(s) in the project. Role(s) should be taken from the [partyRole codelist](http://standard.open-contracting.org/infrastructure/{{version}}/{{lang}}/projects/reference/#party-role). Values from the provided codelist should be used wherever possible, though extended values can be provided if the codelist does not have a relevant code.",
```

Comment:
I think there would be no hyphen in the anchor (`#party-role`), if the heading is constructed the same as the others.

Reply:
fixed in https://github.com/open-contracting/infrastructure/pull/101/commits/a1cbe03fd9f1373507d68f440cc4ed8db5e765bf